crates/host/src/module/wasmer_sys.rs

```diff
@@ -49,18 +49,22 @@ pub fn make_runtime_engine() -> Engine {
     Engine::headless()
 }
 
-/// Take WASM binary and prepare a wasmer Module suitable for iOS
-pub fn build_ios_module(wasm: &[u8]) -> Result<Module, CompileError> {
-    info!(
-        "Found wasm and was instructed to serialize it for ios in wasmer format, doing so now..."
-    );
+/// Compile a wasm binary, serialize it with wasmer's serializtion format, and write to a file.
+/// This file can later be used for contexts where JIT compilation is not possible (iOS for example).
+pub fn write_precompiled_serialized_module_to_file(
```

Comment:
Remove references to iOS in function name and comments, as this functionality can be used in other platforms as well